### DISCUSSION TITLE
Is it possible to display followers/posts from other servers?

### DISCUSSION THREAD

**Discussion by robbietherobot64** [mastodon/mastodon#19936] [Q&A]
I'm a returning mastodon user, exploring the functionality, and one thing I am noticing that I think it going to be a non-starter both for me, as well as many other people, is that you cannot see followers/older posts from other servers.  See the screen shots below for an example of what I mean.

I'm wondering if there is some way for this problem to be overcome, or whether it is a conscious design decision.  I can't see the benefit to hiding older posts or followers, except maybe decreasing the data requirements.  It would be good to be able to see and interact with the posts and followers from other servers, in order to have a seamless experience.  Otherwise I feel like most people are not going to bother clicking around between servers to see the posts/followers for every single situation.  
![image](https://user-images.githubusercontent.com/101053259/200205542-92a98862-ccbd-413d-8179-c5d0c9bead48.png)
![image](https://user-images.githubusercontent.com/101053259/200205594-df0ef4da-c77e-4b5e-ac50-e3e9e170082b.png)

**Comment by trwnh** [comment]
It's less "hiding" information and moreso that the information is not easily obtainable in a consistent way. See #34 and #19880 for example. You can only really be sure of what you have stored locally.
